CONTRIBUTING.md

```diff
@@ -168,8 +168,8 @@ To rebase a feature branch to the latest `main`:
 
    If there were no conflicts, skip this and the next step.
 1. Mark the conflicting files as resolved and continue the rebase
-   ```sh
-   git add *.*
+   ```bash
+   git add .
```

Comment:
From what I understand, this means that `git` will to the file system traversal, rather than the shell, which is probably better due to being OS-independent.